scripts/setup/sha256-tarball-to

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+set -x
+
+SHA256="$1"
+URL="$2"
+shift
+shift
+
+cd /tmp
```

Comment:
Direct use of a world-writable directory tends to be insecure. Consider this pattern to make a new temporary directory (from `install-shellcheck`):

```sh
tmpdir="$(mktemp -d)"
trap 'rm -r "$tmpdir"' EXIT
cd "$tmpdir"
```

Reply:
Thanks for the suggestion!